tests/default/data/dibi.php

```diff
@@ -10,13 +10,15 @@ class Foo
     public function fetch(\Dibi\Connection $connection)
     {
         $row = $connection->fetch('SELECT email, adaid FROM ada where adaid = %i', 1);
-        assertType('array{email: string, adaid: int<-32768, 32767>}|null', $row);
+        assertType('Dibi\Row<array{email: string, adaid: int<-32768, 32767>}>|null', $row);
+        assertType('int<-32768, 32767>', $row->adaid);
```

Comment:
I think to make phpstan aware of the "magic properties" of `Row` we need an additional `PropertiesClassReflectionExtension` for `Row`

https://phpstan.org/developing-extensions/class-reflection-extensions